### DISCUSSION TITLE
Emacs key mappings?

### DISCUSSION THREAD

**Discussion by rdearman** [davidgiven/wordgrinder#242] [Q&A]
Is there anyway to change the default key mappings in Wordgrinder? I use Emacs (a lot) and lots of the key strokes are hardcoded in my hands and they type things without the brain being involved. So I would like to change the key mappings?

Possible, or impossible?

**Comment by davidgiven** [comment]
Go into the Navigation menu and every keystroke is there and rebindable. They'll get saved into your current document. That said, two-key combinations (^X ^C, for example) aren't supported so you can't get the full emacs experience.
